mailpoet/lib/AdminPages/Pages/Newsletters.php

```diff
@@ -121,7 +127,7 @@ public function render() {
 
     $data['sent_newsletters_count'] = $this->newslettersRepository->countBy(['status' => NewsletterEntity::STATUS_SENT]);
     $data['woocommerce_transactional_email_id'] = $this->settings->get(TransactionalEmails::SETTING_EMAIL_ID);
-    $data['display_detailed_stats'] = $this->subscribersFeature->hasValidPremiumKey() && !$this->subscribersFeature->check();
+    $data['display_detailed_stats'] = $this->subscribersFeature->hasValidPremiumKey() && !$this->subscribersFeature->check() && $this->servicesChecker->isPremiumPluginActive();
```

Comment:
Good catch! Would it make sense to highlight this commit in the QA notes so that QA is aware of this fix and can test it if they consider necessary?

Reply:
Good call! I'll update the description.